sky/engine/bindings/scripts/templates/dart_blink.template

```diff
@@ -25,3 +25,14 @@ final Tracing tracing = new Tracing();
 View view;
 
 typedef EventListener(Event event);
+
+/// Linearly interpolate between two numbers.
+num lerpNum(num a, num b, double t) {
```

Comment:
we should probably make this lerpDouble, and not have integers used when calling this.